content/en/guides/core/registry/configure_registry.md

```diff
@@ -39,89 +39,132 @@ A registry admin can remove individual users or entire teams from a registry. To
+| Edit linked artifact description                               | Update           |                   |        |   ✓    |   ✓   |
+| Add or delete a collection's tag                               | Update           |                   |        |   ✓    |   ✓   |
+| Add or delete an alias from a linked artifact                  | Update           |                   |        |   ✓    |   ✓   |
+| Link a new artifact                                            | Update           |                   |        |   ✓    |   ✓   |
+| Edit allowed types list for a registry                         | Update           |                   |        |   ✓    |   ✓   |
+| Edit custom registry name                                      | Update           |                   |        |   ✓    |   ✓   |
+| Delete a collection                                            | Delete           |                   |        |   ✓    |   ✓   |
+| Delete an automation                                           | Delete           |                   |        |   ✓    |   ✓   |
+| Unlink an artifact from a registry                             | Delete           |                   |        |   ✓    |   ✓   |
+| Edit accepted artifact types for a registry                    | Admin            |                   |        |        |   ✓   |
+| Change registry visibility (Organization or Restricted)        | Admin            |                   |        |        |   ✓   |
+| Add users to a registry                                        | Admin            |                   |        |        |   ✓   |
+| Assign or change a user's role in a registry                   | Admin            |                   |        |        |   ✓   |
+
+
+### Effective (inherited) Registry role
+When viewing the registry's membership list, the W&B App shows each user's effective registry role next to the role dropdown in their row.
+
+{{< img src="/images/registry/role_conflict.png" alt="Registry membership list showing the user's effective registry role" >}}
+
+A user's effective (inherited) role in a particular registry matches their _highest_ role among their role in the organization, the registry, and the team that owns the registry, whether inherited or explicitly assigned. For example:
+
+- A team **Admin** or organization **Admin** with the **Viewer** role in a particular registry owned by the team is effectively an **Admin** of the registry.
+- A registry **Viewer** with the **Member** role in the team is effectively a **Member** of the registry.
+- A team **Viewer** with the **Member** role in a particular registry is effectively a **Member** of the registry.
+
+### SDK compatibility
+
+{{% alert title="SDK version requirement" %}}
+To use the W&B SDK to access artifacts as a **Restricted Viewer**, you must use W&B SDK version 0.19.9 or higher. Otherwise, SDK commands will result in permission errors.
```

Comment:
```suggestion
To use the W&B SDK to access artifacts as a **Restricted Viewer**, you must use W&B SDK version 0.19.9 or higher. Otherwise, some SDK commands will result in permission errors.
```